uw-frame-components/portal/messages/partials/notifications-bell.html

```diff
@@ -54,13 +54,13 @@
           <!-- NOTIFICATION CONTENT -->
           <div class="menu-item-content" layout="row" layout-align="space-between center">
             <div flex="80" layout="column" layout-align="center start">
-              <!-- REGULAR MESSAGE -->
-              <span ng-if="!notification.priority">{{ notification.title }}</span>
               <!-- PRIORITY MESSAGE -->
-              <div layout="row" layout-align="start center" ng-if="notification.priority">
-                <md-icon aria-label="important:" flex="10">priority_high</md-icon>
-                <span flex="90"> {{ notification.title }}</span>
+              <div class="menu-item__high-priority" ng-if="notification.priority">
```

Comment:
Maybe should `ng-if` on `notification.priority` having the actual value "high", so that if we would in the future invent other priority values this high priority treatment wouldn't trigger on them.